apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -62,6 +64,8 @@ $(document).ready(function () {
     hiddenLesson,
     verifiedInstructor,
     announcementReducer,
+    progressRedux,
+    isRtl,
```

Comment:
Why are these two needed?  I am not following how these are used in the rest of the PR...